### PR TITLE
chore: update docs with info about multiple CIDR blocks and changes t…

### LINKED ISSUE #3478: [Content Bug] Documentation on VPC peering configuration needs updating as it was not accurate for their setup.
Review https://docs.timescale.com/use-timescale/latest/vpc/ to see what is no longer valid.  Include https://github.com/timescale/docs/issues/3468

### PR DESCRIPTION
**Pull request by billy-the-fish**
Please review VPC Peering and AWS PrivateLink for updates about the pricing plans and CIDR block. If you could ask someone to do a quick validation of the procedure please. A customer  said that they had issues: https://docs-dev.timescale.com/docs-3478-content-bug-documentation-on-vpc-peering-configuration-needs-updating-as-it-was-not-accurate-for-their-setup/use-timescale/3478-content-bug-documentation-on-vpc-peering-configuration-needs-updating-as-it-was-not-accurate-for-their-setup/vpc/

### CODE REVIEW

File: use-timescale/vpc.md [hunk 1]
```diff
@@ -48,69 +47,71 @@ In order to set up VPC peering you need the following permissions in your AWS ac
...
 
 ### Complete the VPC connection in AWS
  
-When you receive the Timescale peering request in AWS, edit your routing table to match 
-the `IP Range` and `CIDR block` between your AWS and Timescale VPCs.
+When you receive the $CLOUD_LONG peering request in AWS, edit your routing table to match 
+the `IP Range` and `CIDR block` between your AWS and $CLOUD_LONG VPCs.
+
+When you peer a VPC with multiple CIDRs, all CIDRs are added to the $CLOUD_LONG rules automatically.
+However, if you change your CIDRs, you have to recreate another peering connection to $CLOUD_LONG.
```
Comment (miqrc) [inline use-timescale/vpc.md]: This point can generate confusion in customers. Please, can you rephrase something like: 
_However, after the peering has been completed, further changes in your VPC's CIDRs won't be detected automatically. If you need to refresh the CIDRs, please recreate the peering connection again._

**Review by esclymeTS — APPROVED** [review]

**Comment by billy-the-fish** [comment]
Hello @miqrc, @esclymeTS , just double-checking before I publish. I had another ticket (https://github.com/timescale/docs/issues/3478) saying that they had issues with the VPC procedure. Did you have time to check that the procedure is correct?

**Comment by miqrc** [comment]
@billy-the-fish I reviewed the doc, and everything seems okay. 
I don't know if it's the issue you refer to, but there was a bug in the code that is already fixed but doesn't affect the procedure.
Also, some users want to peer from GCP/Azure, but that's another story.
